### ISSUE TITLE
Template import/export/duplicate?

### ISSUE THREAD

**Issue by Warpsmith** [woothemes/sensei-certificates#105]
Feature request.

The default certificate template doesn't look very appealing. So it is not unreasonable to want to get a nicer template. But not all are good with graphic design, typography et cetera.
Let's say a professional graphic artist is asked to design a template - as it is now, there's no way to export/import a template - besides possibly the background image.
Could also generate a few extra bucks for you to sell nice templates at woocommerce.com.

Also, it would be nice to be able to duplicate a template, when just some small changes needs to be done.

**Comment by aheckler** [comment]
877502-zen re: duplicating

**Comment by Adminifhta** [comment]
Agrre, would be a very welcome capability

**Comment by jondcampbell** [comment]
Yes, being able to duplicate a template would make so much sense. Having to re-create a new template that is very similar to an existing one is such a pain when it could take only seconds with it built in. And it actually is pretty difficult to get the box positions lined up in the same spot from one certificate to the next, so duplication could help with template consistency.

**Comment by jondcampbell** [comment]
For anyone needing this, I did find that you can create a new template, set the background, save it, then copy the complete meta_value for `_certificate_template` meta_key from the old template you want to copy to the new template in the post_meta table.

**Comment by donnapep** [comment]
Split the import/export functionality into #182.

**Comment by donnapep** [comment]
We should add a _Duplicate_ link when hovering over a certificate template name on the _Certificate Templates_ page:

![Screen Shot 2020-03-27 at 10 10 22 AM](https://user-images.githubusercontent.com/1190420/77764551-312ddc00-7013-11ea-94ec-9c7a3ed211cb.jpg)

When _Duplicate_ is selected, the certificate template is duplicated and has " (Duplicate)" appended to the title.

**Comment by cskindley** [comment]
> It would be nice to be able to duplicate a template, when just some small changes needs to be done.
> 
> See [#105 (comment)](https://github.com/woocommerce/sensei-certificates/issues/105#issuecomment-605022949) for requirements.

Submitted 2016. It’s now Fall of 2023. Jeez

**Comment by rocheston** [comment]
Any solution on how to duplicate certificate templates. Please help. Thank you.

**Comment by mui-tsun** [comment]
2026 and still no ability to duplicate certificates!

**Comment by mui-tsun** [comment]
Hi all. I've just discovered that installing a generic "duplicate post/page" type WordPress plugin will give you the ability to duplicate certificates! The one I've used is by mndpsingh287.
